drivers/client_access_control/spec/requests/client_visibility_legacy_spec.rb

```diff
@@ -11,6 +11,10 @@
 RSpec.describe ClientAccessControl::ClientsController, type: :request do
   include_context 'visibility test context'
 
+  def doc_has_client?(doc, warehouse_client)
+    doc.css("[data-test-client-id='#{warehouse_client.source_id}']").any?
```

Comment:
oh, I see, `test` is for testing....  I guess I still think we should just call it `data-client-id`.